src/foam/nanos/notification/SendEmail.js

```diff
@@ -0,0 +1,19 @@
+foam.CLASS({
+  package: 'foam.nanos.notification',
+  name: 'SendEmail',
+  implements: [ 'foam.nanos.notification.NotificationSetting'],
+  methods: [
+    {
+      name: 'sendNotification',
+      args: [
+        { name: 'x', type: 'Context' },
+        { name: 'notification', type: 'foam.nanos.notification.Notification' },
+      ],
+      javaCode: `
+      return;
+        
+      `
+    }
+  ]
+});
+  
```

Comment:
Why is this model needed? Wouldn't the NotificationConfig implement the NotificationSetting?

Reply:
This is one of the many NotificationSettings in the NotificationConfig.  The NotifcationConfig walks all settings and calls sentNotification()

Reply:
@KateMengjunXia can this be named EmailNotificationSettings?
Also, where is the logic to walk through the settings and call sendNotification?